src/components/v1/ProjectPayoutMods.tsx

```diff
@@ -437,15 +436,15 @@ export default function ProjectPayoutMods({
           }}
           block
         >
-          <Trans>Add a split</Trans>
+          <Trans>Add a payout</Trans>
```

Comment:
Moved the `ProjectPayoutMods` from shared to v1. Also reverted back wording changes that we'd made for v2